magneticfield/inc/Geant/PhysicalConstants.h

```diff
@@ -0,0 +1,115 @@
+
+/**
+ * @brief   Physical constants in internal units.
+ * @file    Geant/PhysicalConstants.h
+ * @author  M Novak, A Ribon
+ * @date    december 2015
+ *
+ * Original file was taken from CLHEP and changed according to GeantV.
+ * Authors of the original version: M. Maire
+ *
+ * The basic units are :
+ * \li  centimeter              (centimeter)
+ * \li  second                  (second)
+ * \li  giga electron volt      (GeV)
+ * \li  positron charge         (eplus)
+ * \li  degree Kelvin           (kelvin)
+ * \li  the amount of substance (mole)
+ * \li  luminous intensity      (candela)
+ * \li  radian                  (radian)
+ * \li  steradian               (steradian)
+ *
+ * You can add your own constants.
+ *
+ */
+
+#ifndef GEANT_PHYSICALCONSTANTS_H
+#define GEANT_PHYSICALCONSTANTS_H
+
+#include "Geant/SystemOfUnits.h"
+
+namespace geant {
+namespace units {
+
+//
+//
+//
+static constexpr double kAvogadro = 6.02214179e+23 / mole;
+
+//
+// kCLight       ->   c   = 2.9979246e+10 [cm/s]
+// kCLightSquare ->   c^2 = 8.9875518e+20 [(cm/s)^2]
+//
+static constexpr double kCLight       = 2.99792458e+8 * m / s;
```

Comment:
Where are `m` and `s` defined? Isn't it dangerous to have such short identifiers globally defined?

Reply:
Well, I know it is not ideal, but they are at least in a namespace: geant::units inside SystemOfUnits.h